app/mobile/src/components/atoms/LargeButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { StyleSheet, TouchableOpacity, Text } from "react-native";
+import { COLORS } from "../../constants/colorPalette";
+
+interface ButtonProps {
+  title: string;
+  onPress: any;
```

Comment:
```suggestion
  onPress: TouchableOpacityProps['onPress'];
```
Better to type the prop fully rather than using `any` if possible, otherwise it limits the type checking TypeScript gives us